airbyte-server/src/test/java/io/airbyte/server/handlers/DestinationDefinitionsHandlerTest.java

```diff
@@ -230,4 +237,12 @@ void testBadFileReceived() throws JsonValidationException, IOException, ConfigNo
 
   }
 
+  private static String LoadIcon(String name) {
+    try {
+      return name == null ? null : MoreResources.readResource("icons/" + name);
+    } catch (IOException e) {
```

Comment:
why not have IOException be part of the method signature like you did on the sources side?

Reply:
Because this is for a test, and if I don't catch it there, it gets out into all the tests that use it.  I figured as long as it's just a test, putting a `try/catch` there is preferable to dealing with checked exception cancer.

Reply:
Could you call the static from the Destination handler and do the try catch then?

Reply:
(also the case isn't standard)